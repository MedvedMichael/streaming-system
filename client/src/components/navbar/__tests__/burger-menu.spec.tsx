import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import BurgerMenu from '../burger-menu'

test ('should render', () => {
    const res = shallow(<BurgerMenu navbarState={false} handleNavbar={() => null}/>)
    expect(toJson(res)).toMatchSnapshot()
})

test ('should have correct className', () => {
    const res = shallow(<BurgerMenu navbarState={true} handleNavbar={() => null}/>)
    expect(!res.find('div.open')).toBe(false)
})