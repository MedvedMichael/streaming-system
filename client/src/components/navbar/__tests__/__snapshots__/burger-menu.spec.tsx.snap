// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<styled.div
  onClick={[Function]}
>
  <div
    className=""
  >
    <span>
       
    </span>
    <span>
       
    </span>
    <span>
       
    </span>
  </div>
</styled.div>
`;
